<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_up</class>
 <widget class="QWidget" name="sign_up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 40, 49);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>540</width>
     <height>250</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(57, 62, 70);
border-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTextEdit" name="textEdit_master_su_pass">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 40, 49);
color: white;
padding-top: 3px;
padding-left: 3px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_master_su_username">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 40, 49);
color: white;
padding-top: 3px;
padding-left: 3px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_up_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(34, 40, 49);
	color: white;
	border: 2px solid rgb(150, 150, 150);
}
QPushButton:hover:!pressed {
   background-color: rgb(150, 150, 150);
}</string>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
    <property name="text">
     <string>SIGN UP</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;
</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_master_su_pass_conf">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 40, 49);
color: white;
padding-top: 3px;
padding-left: 3px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
    <property name="text">
     <string>Confirm Password:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(34, 40, 49);
	color: white;
	border: 2px solid rgb(150, 150, 150);
}
QPushButton:hover:!pressed {
   background-color: rgb(150, 150, 150);
}</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
